arm-network/2017-06-01/swagger/networkInterface.json

```diff
@@ -587,6 +598,20 @@
           "type": "string",
           "description": "The destination port or range."
         },
+        "sourcePortRanges": {
```

Comment:
Per discussion, please check if there are additional breaking changes in this specific API version so that we can avoid releasing multiple major versions of the management libraries. If more changes are coming, we would strongly prefer getting all the changes at once.